src/test/java/studytest/SetTest.java

```diff
@@ -0,0 +1,49 @@
+package studytest;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+/**
+ * Created by owen.ny on 2021/11/04.
+ */
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("Set size 테스트")
```

Comment:
Set size 테스트라는 설명은 이 테스트가 뭘 의미하는지 명확하게 보여주지 않습니다. 
테스트에 대한 설명을 명시할 때는 이 테스트가 어떤 상황에서 어떤 경우 어떤 결과를 보여줄 것인지 명시하는게 좋습니다. 

```suggestion
    @DisplayName("Set 자료구조의 size 메서드는 보유한 요소의 갯수를 반환한다. ")
```